### PR TITLE
Feature/39/add-info-section

### PR DESCRIPTION
**Pull request by GabeDiniz**
The info section is complete for Desktop view.

### CODE REVIEW

File: src/components/index.tsx [hunk 1]
```diff
@@ -14,7 +20,7 @@ export { HeroSection } from "./sections/Hero.section";
 export { FAQSection } from "./sections/FAQ.section";
 
 /*
- * @component Accordioin 
+ * @component Accordioin
```
Comment (juancwu) [inline src/components/index.tsx:23]:
```suggestion
 * @component Accordion
```

Didn't catch this typo before, I think we can fix it up in this PR.

File: src/components/index.tsx [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
 /*
  *
  * @section HeroSection
  *
  */
 export { HeroSection } from "./sections/Hero.section";
 
+/*
+ *
+ * @section FAQSection
```
Comment (juancwu) [inline src/components/index.tsx:10]:
```suggestion
 * @section InfoSection
```

File: src/components/sections/Info.section.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
+const InfoSection = () => {
+  //  00092d
```
Comment (juancwu) [inline src/components/sections/Info.section.tsx:2]: Is this a colour?

**Review by juancwu — CHANGES_REQUESTED** [review]

File: src/components/index.tsx (hunk 1 above)
Comment (GabeDiniz) [reply src/components/index.tsx:23]: Will fix :)

File: src/components/index.tsx (hunk 2 above)
Comment (GabeDiniz) [reply src/components/index.tsx:10]: Nice catch! I must have forgotten to update that. Fixed it

File: src/components/sections/Info.section.tsx (hunk 3 above)
Comment (GabeDiniz) [reply src/components/sections/Info.section.tsx:2]: Yes, and it shouldn't be there. That was temporary! Removed it.

**Review by aidantrabs — APPROVED** [review]

**Comment by aidantrabs** [comment]
Need to fix conflicts

**Pull request merged**